rds/aurora/mysql/cluster/rds.go

```diff
@@ -37,7 +37,7 @@ func NewRDSStack(scope constructs.Construct, props *rds.RDSStackProps) awscdk.St
 	subnetGroup := rds.GetPrivateSubnetGroup(stack, jsii.String("SubnetGroup"), vpc)
 
 	sgs := &[]awsec2.ISecurityGroup{
-		rds.GetAllowAllVPCSecurityGroup(stack, jsii.String("SG"), vpc),
+		rds.GetAllowAllVPCSecurityGroup(stack, jsii.String("SG"), vpc, 3306),
```

Comment:
I refactored this common RDS function to accept the inbound traffic port. MySQL uses 3306 and PostgreSQL uses 5432.